project1.css

```diff
@@ -0,0 +1,80 @@
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-left: 90px;
+}
+
+@media (max-width: 768px) {
+  .professional {
+    background: url(/icons/illastrallation/header-illsutration-mobile.svg);
+  }
+
+  .welcome {
+    width: 90px;
+    height: 28px;
+    font-weight: 600;
+    font-size: 20px;
+    line-height: 28px;
+    align-items: center;
+    text-align: center;
+    color: #28352f;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .hamburger {
+    width: 35px;
+    height: 5px;
+    background-color: black;
+    margin: 6px 0;
+    position: relative;
+  }
+
+  .name {
+    width: 327px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    color: #172b4d;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-left: 15%;
+  }
+
+  .job-title {
+    width: 327px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    color: #174d1a;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-left: 15%;
```

Comment:
- [ ] You should respect to the margins (top, bottom, left, and right) specified in the design, as well as the font and colors (background and font) specified for this template. Please stick to the original vision; it's beautiful and functional, and it will make your job seem fantastic. :wink: :wink: